app/models/checkout.rb

```diff
@@ -10,4 +11,11 @@ class Checkout < ApplicationRecord
   def due_date
     self.created_at + CHECKOUT_PERIOD_IN_DAYS.days
   end
+
+  def fine
+    days_overdue = (Time.zone.now - self.due_date).to_i/1.day
```

Comment:
What do you think about extracting this to a private method? It's not used elsewhere, so it wouldn't be for re-use, more for readability. If I want to know how `fine` works, I don't need to immediately concern myself with how the overdue calculation is done.